netlify.toml

```diff
@@ -55,7 +55,7 @@
 # COMPONENTS/
 [[redirects]]
   from = '/icons/*'
-  to = '/components/icons/:splat'
+  to = '/components/icon/:splat'
```

Comment:
question: we may need another redirect from `/components/icons/*` to `/components/icon/*`

Reply:
Good catch, likely